packages/vapor/scss/redesign/fonts.scss

```diff
@@ -35,6 +35,9 @@
     --big-table-first-row-font-size: #{$big-table-first-row-font-size};
     --big-table-first-row-font-weight: var(--main-font-regular);
     --big-table-second-row-font-size: #{$big-table-second-row-font-size};
+    --flat-select-prepend-font-size: #{$default-font-size};
+    --flat-select-prepend-font-weight: var(--main-font-regular);
+    --badge-font-size: #{$badge-font-size};
```

Comment:
I'm curious, why most variable are now change to using the `var()` css' function, but some don't? (I'm guessing it's because it's a progressive transition, but I've was wondering if in some cases it's not possible to use the css function).

Reply:
CSS variable which is called by `var()` can be overrode in other places, for instance, in`admin-ui`. But variables defined in this file https://github.com/coveo/react-vapor/blob/master/packages/vapor/scss/variables.scss won't change if we don't modify them directly.

Reply:
Gotcha, I've learned today, thanks! :+1: